docs/Technical-Documentation/Acceptance-Criteria-for-Django-admin-console

```diff
@@ -0,0 +1,12 @@
+## Acceptance criteria for dev issues when using Django Admin Console 
+
+The following criteria will be used to determine whether Django admin console can be used to meet security controls in lieu of user interface (UI) for OFA MVP. Django admin console is a backend-only action that a system admin would take. For Django admin console to be used, both of the criteria should be met: 
+
+- Django admin console allows TDRS super admin to modify  account information and access management permissions of TDRS users 
```

Comment:
Please change to "TDP". Also, can you add the note about how the super user admin role gets access (via CLI) per Alex's note in Teams on 12/18?

Reply:
done!